handbook/product/roadmap.md

```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
 
 ## Search
```

Comment:
Not sure if this belongs to the search team (it most likely belong to the web team I think), but there were talks a couple of weeks ago about reducing our cookie footprint to see if we can remove the cookie banner. I think this is important to track, so wanted to mention it here so we don't forget it.

Reply:
@limitedmage  thanks for calling this out. I’ll follow up with the web team (cc: @Joelkw) to figure out where this falls and the scope, and can update this accordingly.

Reply:
@limitedmage @Joelkw fyi opened this issue to track the cookie banner work (unassigned to team for now) [#14724](https://github.com/sourcegraph/sourcegraph/issues/14724)